src/api/storage.js

```diff
@@ -2,14 +2,18 @@
 // Licensed under the MIT license. See LICENSE file in the project root for
 // full license text.
 
+import React from 'react'
+
 /* istanbul ignore file */
 /* eslint-disable no-unused-vars */
 
-export default class Storage {
+export class Storage {
     /**
      * Return an URL for a given media file.
      */
     mediaUrl(name) {
         throw new Error("not implemented")
     }
```

Comment:
This class was only meant to represent an interface, and was only implemented as a class to make `PropTypes.instanceOf` work. Since after this change it will be the default value, this method must not throw but return a usable value (and perhaps log a warning).